package.json

```diff
@@ -12,11 +12,17 @@
     "startAndroidEmulator": "./scripts/start-android.sh",
     "postinstall": "./scripts/postInstall.sh",
     "clean": "npx react-native clean-project-auto",
-    "android": "./scripts/set-pusher-suffix.sh && npx react-native run-android --mode=developmentDebug --appId=com.expensify.chat.dev --active-arch-only",
-    "ios": "./scripts/set-pusher-suffix.sh && npx react-native run-ios --list-devices --mode=\"DebugDevelopment\" --scheme=\"New Expensify Dev\"",
+    "clean-standalone": "./scripts/clean.sh",
+    "android": "./scripts/set-pusher-suffix.sh && ./scripts/run-build.sh --android",
+    "android-standalone": "./scripts/set-pusher-suffix.sh && ./scripts/run-build.sh --android --new-dot",
+    "ios": "./scripts/set-pusher-suffix.sh && ./scripts/run-build.sh --ios",
+    "ios-standalone": "./scripts/set-pusher-suffix.sh && ./scripts/run-build.sh --ios --new-dot",
```

Comment:
Can we do `isHybridApp ? ios : ios-standalone`? So users don't have to choose manually? Or does run-build handle this already?

Reply:
If someone has access to `Mobile-Expensify`, the `run-build` command builds HybridApp by default - for `Expesnify/App` contributors nothing changes, the command will build a standalone NewDot.

I added the `npm run ios-standalone` if a HybridApp dev really wants to build the "original", greenfield version of NewDot. I consider it a bit simpler than invoking the command with some additional flags. I don't think it'll be used frequently, but I wanted to add this option for anyone that wants to do that 😄

Reply:
I'm just checking in here. This seems reasonable, but I want to confirm where the switch logic is for future reference before marking this thread resolved.